mitodl/mitxpro
# Add course topics

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1171 

#### What's this PR do?
Adds `CourseTopic`, updates the courses serializer to render topics, and updates seed data to include topics

#### How should this be manually tested?
Run `./manage.py seed_data`. View the courses API and confirm that you see some course topics
